src/Modal/ModalCard.js

```diff
@@ -33,19 +45,17 @@ const cardBoxStyle = resolve`
  * @param {ModalCard.PropTypes} props
  * @returns {React.Component}
  */
-export const ModalCard = ({ children, small, large }) => {
-    return (
-        <Card
-            className={cx(cardBoxStyle.className, 'modal-card', {
-                small,
-                large,
-            })}
-        >
-            {children}
-            {cardBoxStyle.styles}
-        </Card>
-    )
-}
+export const ModalCard = ({ children, small, large, position }) => (
+    <Card
+        className={cx(cardBoxStyle.className, 'modal-card', position, {
+            small,
+            large,
+        })}
```

Comment:
Perhaps I am overlooking sth, but I don't see how the classList will ever get a `top` class here...

Reply:
Default prop is missing :+1: